### ISSUE TITLE
crate_universe README is out of date

### ISSUE THREAD

**Issue by sayrer** [bazelbuild/rules_rust#782]
I've been trying to write a WORKSPACE using this, which allows a build, but doesn't seem to download any crates...

```
load("@rules_rust//crate_universe:bootstrap.bzl", "crate_universe_bootstrap")
crate_universe_bootstrap()
load("@rules_rust//crate_universe:defs.bzl", "crate_universe", "crate")

crate_universe(
    name = "crate_deps",
    #lockfile = "//tools:crate_universe.lock",
    packages = [
        crate.spec(
            name = "lazy_static",
            semver = "=1.4.0",
        ),
    ],
    supported_targets = [
            "x86_64-apple-darwin",
            "x86_64-unknown-linux-gnu",
    ],
)
```

**Comment by sayrer** [comment]
Thanks to #778 I got this to work:

```
load("@rules_rust//crate_universe:bootstrap.bzl", "crate_universe_bootstrap")
crate_universe_bootstrap()
load("//tools:crate_universe_defaults.bzl", "DEFAULT_URL_TEMPLATE", "DEFAULT_SHA256_CHECKSUMS")
load("@rules_rust//crate_universe:defs.bzl", "crate_universe", "crate")

crate_universe(
    name = "crate_deps",
    #lockfile = "//tools:crate_universe.lock",
    resolver_download_url_template = DEFAULT_URL_TEMPLATE,
    resolver_sha256s = DEFAULT_SHA256_CHECKSUMS,
    packages = [
        crate.spec(
            name = "lazy_static",
            semver = "=1.4.0",
        ),
    ],
    supported_targets = [
            "x86_64-apple-darwin",
            "x86_64-unknown-linux-gnu",
    ],
)

load("@crate_deps//:defs.bzl", "pinned_rust_install")
pinned_rust_install()
```

**Comment by sayrer** [comment]
To be precise, this is the [README](https://github.com/bazelbuild/rules_rust/tree/f7e1652b7d827d27cb23a7a3a4bf55d9b74a19be/crate_universe#readme) file I'm looking at. The `crate_universe:deps.bzl` parts have been deleted, but I wasn't sure how to write a normal WORKSPACE.bazel file from the examples, since they are doing some transformations for the purposes of Bazel CI.

**Comment by sayrer** [comment]
Fixed by #778.
